consent-ui/src/app/auth/login/login.component.spec.ts

```diff
@@ -30,6 +30,8 @@ describe('LoginComponent', () => {
   const usernameInput = 'alex';
   const passwordInput = '1234';
 
+  beforeAll(() => (window.onbeforeunload = jasmine.createSpy()));
```

Comment:
Probably this change and other similar should have been part of some another pull request

Reply:
yes but because my PR failed several times i had to put it there and do it along with this ticket.